README.md

```diff
@@ -1,2 +1,29 @@
 # django-girls-blog-makkaronis4e
 [![Build Status](https://travis-ci.org/kpi-web-guild/django-girls-blog-makkaronis4e.svg?branch=master)](https://travis-ci.org/kpi-web-guild/django-girls-blog-makkaronis4e)
+
+How to run this site from your own computer locally and deploy it on heroku?
+To run locally:
+1. Run your local server
+```(myvenv) ~/djangogirls$ python manage.py runserver```
+2. Print in browser
```

Comment:
Not print, but open. You can also convert this into a clickable link (see Markdown syntax)